keras_imaging/preprocessing/transforms.py

```diff
@@ -0,0 +1,123 @@
+import numpy
+import skimage.transform
+from scipy.ndimage.filters import gaussian_filter
+from scipy.ndimage.interpolation import map_coordinates
+
+"""
+Note: Standardization and transforms assumes that x comes in WxHxC format from the reader
+"""
+
+
+def horizontal_flip(prob):
+    assert 0. < prob < 1.
+
+    def f(x):
+        if numpy.random.random() < prob:
+            return numpy.fliplr(x)
+        return x
+
+    return f
+
+
+def vertical_flip(prob):
```

Comment:
I think we should either drop the probability parameter or have a default of 0.5.